include/aspect/velocity_boundary_conditions/gplates.h

```diff
@@ -403,33 +348,36 @@ namespace aspect
         Tensor<1,2> pointtwo;
 
         /**
-         * Determines the width of the velocity interpolation zone around the
-         * current point. Currently equals the arc distance between evaluation
-         * point and velocity data point that is still included in the
-         * interpolation. The weighting of the points currently only accounts
-         * for the surface area a single data point is covering ('moving
-         * window' interpolation without distance weighting).
+         * Determines the depth of the lithosphere. The user might want to apply
+         * the GPlates velocities not only at the surface of the model, but also
+         * in the whole lithosphere.
```

Comment:
Explain a bit more what is happening there. Something like: At every side boundary point with a depth smaller than this value the surface velocity will be prescribed.